libfreerdp/codec/color.c

```diff
@@ -441,7 +444,7 @@ BOOL freerdp_image_copy(BYTE* pDstData, DWORD DstFormat,
 			/* Copy down */
 			if (nYDst < nYSrc)
 			{
-				for (y = 0; y < nHeight; y++)
+				for (y = 0; y < (INT32)nHeight; y++)
```

Comment:
Because in some loops we count down from top to bottom and abort condition is `-1` for the iterator. Since we have already proven that the value can fit in an `INT32` the casts were the easier choice.